templates/CommentsInterface_singlecomment.ss

```diff
@@ -34,10 +34,12 @@
 				<% end_if %>
 			</div>
 			<% if $RepliesEnabled %>
-				<a class="comment-reply-link" href="#{$ReplyForm.FormName}">Reply to $AuthorName.XML</a>
+                    <a data-comment-id="$ID" class="comment-reply-link" href="$Parent.Link?replyTo=$ID#Form_ReplyForm_$ID"><% _t('CommentsInterface_singlecomment_ss.REPLY_TO','Reply to') %>&nbsp;$AuthorName.XML</a>
```

Comment:
That one looks like an error on my part but I have had to manually edit some files due to a mix of tabs and spaces.  Conversion to PSR2 would resolve this.  Will raise it as an issue.